go.mod

```diff
@@ -1,16 +1,17 @@
 module github.com/dave/dst
```

Comment:
There seems to be a lot more changes to `go.mod` than I'd expect... Can we revert them without breaking things?

Reply:
You are 100% correct. I was fumbling around early on in my work and didn't realize how everything worked together. I think that I have it cleaned up now so that the change to these files are not required. Of course, I cannot test outside of what I have installed, so please let me know if I did something wrong!